channels/fast-4.11.yaml

```diff
@@ -0,0 +1,6 @@
+feeder:
+  delay: PT0H
+  filter: 4[.](10|11)[.][0-9].*
+  name: fast
+name: fast-4.11
+versions:
```

Comment:
We need to define versions as an array : versions: [] 
example : https://github.com/openshift/cincinnati-graph-data/commit/3b5b5436880d84c674b27bdbffffced7863f5f86